tox.ini

```diff
@@ -20,13 +21,15 @@ commands =
 
 [testenv:type]
 skipsdist = true
-skip_install = true
 deps =
     mypy
     django-stubs
     djangorestframework-stubs
+    types-setuptools
```

Comment:
```suggestion
```

We shouldn't need this. We're not checking `setup.py`, we're looking directly inside `dandiapi/`.

Reply:
We're using `setuptools_scm` for versioning in this file https://github.com/dandi/dandi-archive/blob/master/dandiapi/__init__.py. We could ignore it, but I don't think it hurts that much either to add it as a dependency. :shrug: I added it back as a `dev` extra dependency, but I'm fine with just excluding this one file too if installing `types-setuptools` is too overkill - any thoughts @brianhelba @AlmightyYakob ?

Reply:
I didn't realize you had that import of `pkg_resources` in your actual project module. I think it's correct to include `types-setuptools` for that. Sorry for my confusion.

@AlmightyYakob This is totally out of scope of this PR, but note there's a way to get the package version from Python 3.8+ standard libraries (rather than setuptools): https://github.com/pypa/setuptools_scm/#retrieving-package-version-at-runtime